engine/engine.go

```diff
@@ -150,12 +156,20 @@ type FileInstance struct {
 	FS   string `json:"fs"`
 }
 
+func list2Map(list []string) map[string]bool {
+	ret := map[string]bool{}
```

Comment:
`make(map[string]bool, len(list))` is usually used in case you know the final size of the map, this causes not reallocating the map any time it exceeds it's capacity.

Reply:
Nice